pkg/reconciler/kubernetes/tektonresult/tektonresult.go

```diff
@@ -145,12 +154,24 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tr *v1alpha1.TektonResul
 		return errors.New(errMsg)
 	}
 
-	// check if the secrets are created
-	// TODO: Create secret automatically if they don't exist
-	// TODO: And remove this check in future release.
-	if err := r.validateSecretsAreCreated(ctx, tr); err != nil {
-		return err
+	// If external database is disable then create default database and tls secret
```

Comment:
If the external database is disabled, create a default database and a TLS secret.
Otherwise, verify if the default database secret is already created, and ensure the TLS secret is also created.

Reply:
Are we sure that the code below creates a database ? it seems jsut creating creds

Reply:
> Are we sure that the code below creates a database ? it seems jsut creating creds

yes, it creates the database